doc/api-extensions.md

```diff
@@ -240,6 +240,12 @@ will look for an existing interface for that VLAN on the host. If one
 can't be found it will create one itself and then use that as the
 macvlan parent.
 
+## network\_gvrp
+This adds an optional `gvrp` property to `macvlan` `ipvlan` `routed` and `physical` network devices.
```

Comment:
@TimRots when you add this commit to https://github.com/lxc/lxd/pull/8325 please can you change this message to be:

```
This adds an optional `gvrp` property to `macvlan` and `physical` networks, and to `ipvlan`, `macvlan`, `physical`, and `routed`  NIC devices.
```

In LXD the managed networks and the NIC devices are different concepts, so important to differentiate them in the docs.